molgenis-data-import/src/main/java/org/molgenis/data/importer/EmxMetaDataParser.java

```diff
@@ -1,51 +1,59 @@
 package org.molgenis.data.importer;
 
-import com.google.common.collect.*;
+import com.google.common.collect.ImmutableMap;
```

Comment:
yes I applied the optimisation. You only use the ImmutableMap from collect package + a handful of static imports so its not necessary to import the entire google.common.collect package